src/Workspaces/Core/Portable/Workspace/Host/Mef/MefHostServices.cs

```diff
@@ -13,6 +13,10 @@ namespace Microsoft.CodeAnalysis.Host.Mef
 {
     public class MefHostServices : HostServices, IMefHostExportProvider
     {
+        internal delegate MefHostServices CreationHook(IEnumerable<Assembly> assemblies, bool requestingDefaultHost);
+
+        private static CreationHook s_CreationHook;
```

Comment:
Ditto: comment this is test only. Even better: name the field appropriately.

Reply:
➡️ Added comments in 37a1d2c